snuba/snuba_migrations/replays/0009_add_dom_index_columns.py

```diff
@@ -0,0 +1,103 @@
+from typing import Iterator, Sequence, Tuple
+
+from snuba.clickhouse.columns import Array, Column, String, UInt
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+
+columns: Sequence[Tuple[Column[Modifiers], str]] = [
+    (Column("click_node_id", UInt(32, Modifiers(default="0"))), "tags.value"),
+    (
+        Column("click_tag", String(Modifiers(default="''", low_cardinality=True))),
+        "click_node_id",
+    ),
+    (Column("click_id", String(Modifiers(default="''"))), "click_tag"),
+    (Column("click_class", Array(String())), "click_id"),
+    (Column("click_text", String(Modifiers(default="''"))), "click_class"),
+    (
+        Column("click_role", String(Modifiers(default="''", low_cardinality=True))),
+        "click_text",
+    ),
+    (Column("click_alt", String(Modifiers(default="''"))), "click_role"),
+    (Column("click_testid", String(Modifiers(default="''"))), "click_alt"),
+    (Column("click_aria_label", String(Modifiers(default="''"))), "click_testid"),
+    (Column("click_title", String(Modifiers(default="''"))), "click_aria_label"),
+]
+
+alters: Sequence[str] = ["title", "user", "sdk_name", "sdk_version"]
```

Comment:
Can you split these changes out into a separate PR? They are causing the test failures.

The test failures are because the existing columns in the storage configuration don't match what is in CH. You'll have to change the storage configuration to have the correct type, but that will muddy this PR up even more.